mmdet/apis/train.py

```diff
@@ -162,6 +162,9 @@ def build_optimizer(model, optimizer_cfg):
         bias_lr_mult = paramwise_options.get('bias_lr_mult', 1.)
         bias_decay_mult = paramwise_options.get('bias_decay_mult', 1.)
         norm_decay_mult = paramwise_options.get('norm_decay_mult', 1.)
+        no_wd_in_dw = paramwise_options.get('no_wd_in_dw', False)
```

Comment:
I suggest renaming this option to `dwconv_decay_mult` and use it as a multiplier.